sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlockBlobApiTests.java

```diff
@@ -1797,6 +1800,50 @@ public void uploadAndDownloadRegionLengthWithDiffBlobClients(int regionLength, i
         assertArraysEqual(data.array(), plaintextOut.toByteArray());
     }
 
+    @Test
+    public void validateContentEncryptionKeyProtocol() throws IOException {
+        bec = getEncryptionClient(EncryptionVersion.V2_1, generateBlobName());
```

Comment:
Let's parameterize this case to do both V2 and V2_1 to make sure we use the correct versions when embedding into the content key